src/parser/utils.rs

```diff
@@ -107,6 +107,25 @@ pub fn elaborate_field_path(
     (fst, content)
 }
 
+
+pub fn build_letblock<I>(fields: I, rest: RichTerm) -> Term
+where
+    I: IntoIterator<Item = (Ident, RichTerm)> + Debug,
```

Comment:
```suggestion
    I: IntoIterator<Item = (Ident, RichTerm)>,
```

I think the `Debug` thing is a leftover from development in the `build_record`. If it compiles without, let's get rid of it